### PR TITLE
added iTunes

### PR DESCRIPTION
**Pull request by cfoellmann**
Current status: __Experimental__

### CODE REVIEW

File: pending/itunes.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  <description>Media Player</description>
+  <option id="cache">
+    <label>Cache</label>
+    <description>Delete the cache</description>
+    <action search="file" command="delete" path="$LocalAppData\Apple Computer\iTunes\Cache.db"/>
+  </option>
+  <option id="cache_artwork">
+    <label>Album Artwork Cache</label>
+    <description>Delete the cache</description>
+    <action search="walk.all" command="delete" path="$USERPROFILE\My Music\iTunes\Album Artwork\Cache"/>
```
Comment (cfoellmann) [inline pending/itunes.xml]: @az0 `$USERPROFILE\My Music` doesn't work herer. The windows explorer correctly redirects to my moved folder but bleachbit cannot follow.
I found https://github.com/az0/bleachbit/blob/96500aafb38c0539159b764a53be1fef55988762/bleachbit/Windows.py#L542 . How can I use the decleration of 'music'? `$MUSIC` and `$MYMUSIC` don't work.

**Comment by nodiscc** [comment]
@cfoellmann hi, does this cleaner currently work?

**Comment by nodiscc** [comment]
@cfoellmann ping

**Comment by nodiscc** [comment]
No info provided for 6 months, closing. Feel free to reopen if you have updates on this.

**Pull request closed without merging**